src/Symfony/Component/HttpKernel/Tests/EventListener/RouterListenerTest.php

```diff
@@ -104,9 +104,6 @@ public function testSubRequestWithDifferentMethod()
                        ->will($this->returnValue(array()));
 
         $context = new RequestContext();
-        $requestMatcher->expects($this->any())
-                       ->method('getContext')
-                       ->will($this->returnValue($context));
```

Comment:
This was useless as the `getContext()` method will only be called when no request context was passed to the constructor of the `RouterListener`